diskfs_unix.go

```diff
@@ -1,5 +1,5 @@
-//go:build !darwin || linux || solaris || aix || freebsd || illumos || netbsd || openbsd || plan9
-// +build !darwin linux solaris aix freebsd illumos netbsd openbsd plan9
+//go:build linux || solaris || aix || freebsd || illumos || netbsd || openbsd || plan9
+// +build linux solaris aix freebsd illumos netbsd openbsd plan9
```

Comment:
Why are we removing `!darwin` from this list?

Reply:
There are 2 issues with `!darwin` in the ``go:build`` condition:
1. It is not needed to explicitly exclude `darwin` since nothing in the condition matches it
2. As previously written, it reads "build for this list of platforms or anything that is not `darwin`" which also matches... windows :smile_cat: 

Regarding the "+build", this is a Go 1.16 equivalent of `go:build`.